src/views/Comments.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable no-shadow */
+import React, { Component } from "react";
+import Textarea from "react-textarea-autosize";
+import PropTypes from "prop-types";
+import moment from "moment";
+import authorImage from "../assets/img/user.jpg";
```

Comment:
can you use no-image `no-thumbnail.png` as an indicator that there is no user-avatar?

Reply:
or use `user.png` as default user-avatar.